redpen-core/src/main/java/cc/redpen/RedPen.java

```diff
@@ -138,7 +139,6 @@ public Document parse(DocumentParser parser, String content) throws RedPenExcept
     public Map<Document, List<ValidationError>> validate(List<Document> documents) {
         Map<Document, List<ValidationError>> docErrorsMap = new HashMap<>();
         documents.forEach(e -> docErrorsMap.put(e, new ArrayList<>()));
-        initializeValidators();
```

Comment:
I've tried that and it seems to work okay - the pull request now reflects this change.